package.json

```diff
@@ -26,43 +26,44 @@
     "url": "https://github.com/rangle/angular-2-aot-sandbox/issues"
   },
   "dependencies": {
```

Comment:
I think it is better to always use the newest deps so that users may find invalid old cases and report it. After all, this repo is to find what best AoT can do :)